easybuild/easyblocks/i/icc.py

```diff
@@ -60,13 +61,11 @@ class EB_icc(IntelBase):
         - will fail for all older versions (due to newer silent installer)
     """
 
-    def configure_step(self):
-        """have fallback for components for > 2016 versions"""
+    def __init__(self, *args, **kwargs):
+        """Constructor, initialize class variables."""
+        super(EB_icc, self).__init__(*args, **kwargs)
 
-        if LooseVersion(self.version) >= LooseVersion('2016') and not self.cfg['components']:
```

Comment:
As long as you pick the right set of components, you should be fine?

Trying to specify components shouldn't be a reason to raise a red flag imho, even for older versions (we know that the 2015 versions are also too fat by default, for example).